piqe_ocp_lib/api/resources/ocp_limit_ranges.py

```diff
@@ -0,0 +1,160 @@
+            kind: limitrange
+            name: test
+            uid: aa1a8359-42b0-45f9-b44b-cd0ecbff6ef8
+          kind: Status
+          metadata: {}
+          status: Success
+
+        """
+        api_response = None
+        try:
+            api_response = self.ocp_limit_range.delete(namespace)
+        except ApiException as e:
+            logger.exception(f"Exception while deleting LimitRanges: {e}\n")
+        return api_response
+
+    def create_a_limit_range(self, limit_ranges_body):
+        """
+        Create a LimitRanges in specific namespace
+        :param limit_ranges_body (dict) LimitRanges definitions
+        :return: CreateLimitRanges response on success OR None on Failure
+        """
+        create_lr_response = None
+        try:
+            create_lr_response = self.ocp_limit_range.create(body=limit_ranges_body)
+        except ApiException as e:
+            logger.exception(f"Exception while creating LimitRanges definitions : {e}\n")
+
+        return create_lr_response
+
+    def get_a_limit_range(self, name, namespace):
+        """
+        Get a specific LimitRanges from specific namespace
+        :param name: (str) name of LimitRanges
+        :param namespace: (str) namespace where LimitRanges is created
+        :return: LimitRanges response on success OR None on Failure
+        """
+        api_response = None
+        try:
+            api_response = self.ocp_limit_range.get(name=name, namespace=namespace)
+        except ApiException as e:
+            logger.error(f"Exception while getting a {name} LimitRanges : {e}\n")
+
+        return api_response
+
+    def delete_a_limit_range(self, name, namespace):
+        """
+        Delete a specified LimitRange from specified namespace
+        :param name: (str) name of LimitRange
+        :param namespace: (str) name of namespace where LimitRange was created
+        :return: Delete response on success OR None on Failure
+
+        ResourceInstance[Status]:
+          apiVersion: v1
+          details:
+            kind: limitrange
+            name: test
+            uid: aa1a8359-42b0-45f9-b44b-cd0ecbff6ef8
+          kind: Status
+          metadata: {}
+          status: Success
+
+        """
+        api_response = None
+        try:
+            api_response = self.ocp_limit_range.delete(name, namespace)
+        except ApiException as e:
+            logger.exception(f"Exception while deleting {name} LimitRanges: {e}\n")
+        return api_response
+
+    def update_a_limit_range(self, namespace, name, **kwargs):
+        """
+        Method to change the limit ranges for a deployment
+        :param namespace: The namespace containing the targeted
+                          deployment config
+        :param name: The targeted limit range
+        :param kw: either a list of dicts or a dict containing the type and max/min ranges
+        :return: A V1LimitRange object
+        """
+        body = {'spec': {'limits': kwargs.get('body')}}
```

Comment:
We might want to look into possibly breaking this down to have separate methods for different types.